docs/articles/documentation/guides/advanced-guides/multiple-browser-windows.md

```diff
@@ -167,6 +169,15 @@ Use one of the following settings to disable support for multiple browser window
 
 This section describes the limitations of multiple browser windows testing in TestCafe.
 
+### Supported Browsers
+
+TestCafe's multi-window mode supports the following browsers:
+
+* Chrome (headless and headed)
+* Firefox (headless and headed)
+
+Multiple browser windows are not available in cloud, remote browsers and during device emulation.
```

Comment:
Add links to the specified browser providers and specify that it's Chrome's device emulation.

Something like this:
```js
Multiple browser windows are not available in [cloud](https://devexpress.github.io/testcafe/documentation/guides/concepts/browsers.html#browsers-in-cloud-testing-services), ... and with [Chromium device emulation](https://devexpress.github.io/testcafe/documentation/guides/concepts/browsers.html#use-chromium-device-emulation).
```